### PR TITLE
- Added ability to see progress to weight goal.

### PR DESCRIPTION
**Pull request by yuxianglim**
- Fixed bug on set-profile whereby past weight history is not reset
- Minor changes to PPP
- added ability to see the progress to weight goal in check-weight-progress command

### CODE REVIEW

File: docs/team/yuxianglim.md [hunk 1]
```diff
@@ -18,14 +18,23 @@ health data and weight goal.
 ### Code Contributed
 **__Diet Manager__**:<br>
 Link to code on tP Code Dashboard 
-[**__here__**](https://nus-cs2113-ay1920s2.github.io/tp-dashboard/#breakdown=true&search=lamyuewei&sort=groupTitle&sortWithin=title&since=2020-03-01&timeframe=commit&mergegroup=false&groupSelect=groupByRepos).
+[**__here__**](https://nus-cs2113-ay1920s2.github.io/tp-dashboard/#breakdown=true&search=yuxianglim&sort=groupTitle&sortWithin=title&since=2020-03-01&timeframe=commit&mergegroup=false&groupSelect=groupByRepos).
```
Comment (lamyuewei) [inline docs/team/yuxianglim.md:21]: Good job identifying the right name to use, ahahah

File: src/main/java/seedu/dietmanager/logic/commands/profile/CheckWeightRecordCommand.java [hunk 2]
```diff
@@ -66,12 +68,31 @@ public Result getResult(Profile profile) {
...
+            }
         } else {
-            this.resultString = String.format(MessageBank.WEIGHT_GAIN_MESSAGE, abs(weightDifference));
+            if (currentWeight == weightGoal) {
+                this.resultString = String.format(MessageBank.WEIGHT_GAIN_MESSAGE
+                        + MessageBank.WEIGHT_GOAL_ACHIEVED_MESSAGE, abs(weightDifference), weightToGoal);
+            } else {
+                this.resultString = String.format(MessageBank.WEIGHT_GAIN_MESSAGE
+                        + MessageBank.WEIGHT_GOAL_NOT_ACHIEVED_MESSAGE, abs(weightDifference), weightToGoal);
+            }
```
Comment (lamyuewei) [inline src/main/java/seedu/dietmanager/logic/commands/profile/CheckWeightRecordCommand.java:95]: This portion is a bit long, maybe afterward can try to extract the method. Or split it up into different cases

File: src/main/java/seedu/dietmanager/logic/commands/profile/CheckWeightRecordCommand.java [hunk 3]
```diff
@@ -66,12 +68,31 @@ public Result getResult(Profile profile) {
...
         }
 
         return new Result(this.resultString);
```
Comment (lamyuewei) [inline src/main/java/seedu/dietmanager/logic/commands/profile/CheckWeightRecordCommand.java:98]: Good job in adding this check weight portion

**Review by lamyuewei — COMMENTED** [review]
Good job for making these changes

**Comment by lamyuewei** [comment]
There are some 2 checks that fail, do try to resolve them

File: docs/team/yuxianglim.md (hunk 1 above)
Comment (yuxianglim) [reply docs/team/yuxianglim.md:21]: Thanks haha i mean if i use yours i will get more code right ahaah

**Comment by lamyuewei** [comment]
All checks passed, good to merge. Approved

**Review by lamyuewei — APPROVED** [review]
Approved, good job

**Pull request merged**